tests/test_errenskitchen.py

```diff
@@ -0,0 +1,79 @@
+# mypy: allow-untyped-defs
+
+from recipe_scrapers.errenskitchen import ErrensKitchen
+from tests import ScraperTest
+
+
+class TestErrensKitchenScraper(ScraperTest):
+    scraper_class = ErrensKitchen
+
+    def test_host(self):
+        self.assertEqual("errenskitchen.com", self.harvester_class.host())
+
+    def test_author(self):
+        self.assertEqual("Erren Hart", self.harvester_class.author())
+
+    def test_category(self):
+        self.assertEqual("Dinner", self.harvester_class.category())
+
+    def test_cuisine(self):
+        self.assertEqual("Italian", self.harvester_class.cuisine())
+
+    def test_description(self):
+        self.assertEqual(
+            "A delectable dish of Creamy Sun-Dried Tomato Pasta with Chicken flavored with Parmesan cheese, sun-dried tomatoes, onions, and garlic.",
+            self.harvester_class.description(),
+        )
+
+    def test_image(self):
+        self.assertEqual(
+            "https://www.errenskitchen.com/wp-content/uploads/2022/05/Chicken-Sundried-Tomato-Pasta-06.jpg",
+            self.harvester_class.image(),
+        )
+
+    def test_ratings(self):
+        self.assertEqual(5.0, self.harvester_class.ratings())
+
+    def test_title(self):
+        self.assertEqual(
+            "Creamy Sun-Dried Tomato Pasta with Chicken", self.harvester_class.title()
+        )
+
+    def test_total_time(self):
+        self.assertEqual(30, self.harvester_class.total_time())
+
+    def test_yields(self):
+        self.assertEqual("4 servings", self.harvester_class.yields())
+
+    def test_ingredients(self):
+        expected_ingredients = [
+            "1 tablespoon olive oil",
+            "2 lbs boneless chicken thighs (or breasts, cubed)",
+            "1 onion (chopped)",
+            "¼ cup sun-dried tomatoes (drained and chopped)",
+            "2 tablespoons sun-dried tomato paste",
+            "3 garlic cloves (minced)",
+            "¾ cup chicken stock",
+            "¾ cup freshly grated Parmesan cheese",
+            "1½ cups half and half (or whipping cream)",
+            "1 pound pasta",
+            "1 tablespoon fresh basil (chopped)",
+            "Salt and pepper to taste",
+        ]
+        self.assertEqual(expected_ingredients, self.harvester_class.ingredients())
+
+    def test_instructions(self):
+        expected_instructions = "\n".join(
```

Comment:
This is still messier than if `instructions()` returned a `list[str]` natively.. but I think it's an improvement over repeating the `\n` escapes within the expected-instruction structure.